yt/data_objects/particle_trajectories.py

```diff
@@ -137,10 +139,16 @@ def __init__(self, outputs, indices, fields=None, suppress_logging=False):
 
     def has_key(self, key):
         return (key in self.field_data)
-    
+
     def keys(self):
         return self.field_data.keys()
 
+    def get_full_field_name(self, field):
+        ds_first = self.data_series[0]
+        dd_first = ds_first.all_data()
+        ptype = self.ptype if self.ptype else 'all'
+        return dd_first._determine_fields((ptype, field))
```

Comment:
Do we have to do `_determine_fields` on an object or can we just do something like `ds._get_field_info`? @ngoldbaum ?

Reply:
Either will work and give the same answer.

Reply:
This is fine as-is I think.